Letterbook.Workers/Consumers/DeliveryWorker.cs

```diff
@@ -0,0 +1,38 @@
+using Letterbook.Core;
+using Letterbook.Core.Adapters;
+using Letterbook.Core.Models;
+using Letterbook.Workers.Contracts;
+using MassTransit;
+
+namespace Letterbook.Workers.Consumers;
```

Comment:
Git decided this is a new file, rather than a rename, which is unfortunate. This and the other masstransit workers are just updates of workers that used to be in core.